### PR TITLE
add a pull request template (and mention the CLA)

### PR DESCRIPTION
**Pull request by jlipps**

### CODE REVIEW

**Comment by jlipps** [comment]
@bootstraponline @imurchie please review

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+## Proposed changes
+
+Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
+
+## Types of changes
+
+What types of changes does your code introduce to Appium?
+_Put an `x` in the boxes that apply_
+
+- [] Bugfix (non-breaking change which fixes an issue)
```
Comment (imurchie) [inline .github/PULL_REQUEST_TEMPLATE.md]: I think all these checkboxes need a space in between the square brackets to render correctly.

**Comment by jlipps** [comment]
@imurchie fixed and added a couple bits as well

**Comment by imurchie** [comment]
Looks good to me.

**Pull request merged**